rest_framework_nested/relations.py

```diff
@@ -103,4 +96,4 @@ def __init__(self, view_name=None, **kwargs):
         assert view_name is not None, 'The `view_name` argument is required.'
         kwargs['read_only'] = True
         kwargs['source'] = '*'
-        super(NestedHyperlinkedIdentityField, self).__init__(view_name=view_name, **kwargs)
+        super().__init__(view_name=view_name, **kwargs)
```

Comment:
That part can stay. It helps typing tools.
```suggestion
        super(NestedHyperlinkedIdentityField, self).__init__(view_name=view_name, **kwargs)
```